js/src/html/beautifier.js

```diff
@@ -182,6 +184,14 @@ function Beautifier(html_source, options, js_beautify, css_beautify) {
                     }
                 }
                 return false;
+            },
+            get_tag_name: function(full_tag) {
```

Comment:
This information already found during the process of processing the tag: https://github.com/madman-bob/js-beautify/blob/def3cbda7911a343ff603b81a03d63670db456a4/js/src/html/beautifier.js#L534

It would be better to store or pass that information back form there.

Reply:
The linked line only gets the tag name, while this function also distinguishes whether it's an opening/closing tag. Ideally, I'd refactor the other locations to use this function, but that would introduce minor behavioural changes.